src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -702,6 +702,14 @@ public override void VisitCoalesce(ICoalesceOperation operation)
             var valueConversion = operation.ValueConversion;
         }
 
+        public override void VisitCoalesceAssignment(ICoalesceAssignmentOperation operation)
+        {
+            Assert.Equal(OperationKind.CoalesceAssignment, operation.Kind);
+            Assert.NotNull(operation.Target);
+            Assert.NotNull(operation.Value);
```

Comment:
These are probably redundant. Elsewhere we check that all items in ```operation.Children``` are not null. In combination with that, the next assert ensures Target and Value are not null.